staging/src/k8s.io/client-go/tools/record/events_cache_test.go

```diff
@@ -234,7 +234,7 @@ func TestEventCorrelator(t *testing.T) {
 
 	for testScenario, testInput := range scenario {
 		eventInterval := time.Duration(testInput.intervalSeconds) * time.Second
-		clock := clock.IntervalClock{Time: time.Now(), Duration: eventInterval}
+		clock := simpleintervalclock.SimpleIntervalClock{Time: time.Now(), Duration: eventInterval}
```

Comment:
stuttering here too

Reply:
ah woops sorry about that, this must've creeped in while reverting the previous change. Thanks!